lib/generators/shopify_app/install/templates/shopify_app_ready_script.html

```diff
@@ -0,0 +1,8 @@
+<script type="text/javascript">
+  ShopifyApp.ready(function(){
+    ShopifyApp.Bar.initialize({
+      title: "Home",
+      icon: "<%= asset_path('faveicon.png') %>"
```

Comment:
this probably shouldn't be here - we used to copy over some assets including this png with the old generator. Options - add this png to the new generator or remove this line of code.

nice catch!